src/Sentry.Unity/UnitySentryOptions.cs

```diff
@@ -1,9 +1,12 @@
 using System;
 using System.IO;
+using System.Runtime.CompilerServices;
 using System.Text.Json;
 using Sentry.Extensibility;
 using UnityEngine;
 
+[assembly: InternalsVisibleTo("Sentry.Unity.Editor")]
```

Comment:
We already have this here:

https://github.com/getsentry/sentry-unity/blob/fd4a26e239d18865f1c947e3551178a1cde00120/src/Sentry.Unity/Properties/AssemblyInfo.cs#L3